features/path-shape.yml

```diff
@@ -0,0 +1,10 @@
+name: "`path()` shape"
```

Comment:
Let's do this feature in a separate PR (from synchronous discussion)

Reply:
To elaborate here: the BCD data is very messy. It has subfeatures to describe support in `d`, `shape-outside`, and `offset-path`, but _also_ represents that information as partial support on `path`. It ought to do one or the other. We'll need to fix this in BCD.

(Please @ me on that PR, when you open it.)

Reply:
Moved to https://github.com/web-platform-dx/web-features/pull/1799